cmd/process-agent/command/main_common.go

```diff
@@ -165,6 +165,8 @@ func runApp(ctx context.Context, globalParams *GlobalParams) error {
 			profiler.Component,
 			expvars.Component,
 			apiserver.Component,
+			// TODO: This is needed by the container-provider and should be updated to be handled by it
```

Comment:
Container provider is not currently a component, but when it's converted, we should ensure the tagger is a dependency